Pharmaceuticals proofreading (#993)
greenelab/covid19-review: content/20.pharmaceuticals.md

To assess the efficacy and safety of bamlanivimab alone or in combination with etesevimab for the treatment of COVID-19, a phase 2/3 trial (BLAZE-1) [@clinicaltrials:NCT04427501] was initiated.
The interim analysis of the phase 2 portion suggested that bamlanivimab alone was able to accelerate the reduction in viral load [@doi:10.1056/NEJMoa2029849].
However, more recent data suggests that only the bamlanivimab/etesevimab combination therapy is able to reduce viral load in COVID-19 patients [@doi:10.1001/jama.2021.0202].

> Based on this data, the combination therapy received an EUA for COVID-19 from the FDA in February 2021 [@url:https://www.fda.gov/media/145801/download].

Author: I had removed the "of".  We didn't use it with dates elsewhere.

Six leads were identified, and _in vitro_ analysis revealed that ebselen, an organoselenium compound with anti-inflammatory and antioxidant properties [@doi:10.1007/s11033-014-3417-x], had the strongest potency in reducing the viral load in SARS-CoV-2-infected Vero cells [@doi:10.1038/s41586-020-2223-y].
Ebselen is an organoselenium compound with anti-inflammatory and antioxidant properties [@doi:10.1007/s11033-014-3417-x]; however, it is likely to be a promiscuous binder, which could diminish its therapeutic potential [@doi:10.1038/s41586-020-2223-y].
While there is clear computational and _in vitro_ support for ebselen's potential as a COVID-19 therapeutic, results from clinical trials are not yet available for this compound.
In July 2020 phase II clinical trials commenced to assess the effects of SPI-1005, an investigational drug from Sound Pharmaceuticals that contains ebselen [@url:https://www.contractpharma.com/contents/view_breaking-news/2020-08-31/fda-clears-spis-ebselen-for-phase-ii-covid-19-trials], on 60 adults presenting with each of moderate [@clinicaltrials:NCT04484025] and severe [@clinicaltrials:NCT04483973] COVID-19.

> Another candidate has emerged from a broader assessment of work in computational prediction of candidate drugs for repurposing against COVID-19.

Reviewer: Not part of your changes, but I found this sentence quite confusing. Candidate sounds like it refers to a drug candidate, but no specific one is mentioned in the paragraph. Maybe `Another candidate -> Other candidates`?

These resources were integrated into the COVID-19 Drug and Gene Set Library, which at the time of its publication contained 1,620 drugs sourced from 173 experimental and computational drug sets and 18,676 human genes sourced from 444 gene sets [@doi:10.1016/j.patter.2020.100090].
This study found that several drugs had been identified as candidates across multiple independent analyses, including several that had received significant attention as potential therapeutics for COVID-19 (e.g., CQ/HCQ and remdesivir) [@doi:10.1016/j.patter.2020.100090].
A second study expanded on these results by identifying that cationic amphiphilicity was a shared property among many of the candidate drugs identified through both computational and phenotypic screens, suggesting it as a potential cause of a compound's proposed antiviral activity [@doi:10.1126/science.abi4708].
Cationic amphiphilic drugs are known to initiate phospholipidosis, which is when phospholipids accumulate in the lysosome [@doi:10.1515/hsz-2019-0270].
Mechanistically, phospholipidosis could disrupt viral replication by inhibiting lipid processing [@doi:10.1080/01913123.2020.1850966] (see discussion of HCQ in Appendix A).

> They found that antiviral activity against SARS-CoV-2 was correlated with phospholipidosis for drugs with and without cationic amphiphilicity.

Reviewer 2: ```suggestion
They found that antiviral activity against SARS-CoV-2 _in vitro_ was correlated with phospholipidosis for drugs with and without cationic amphiphilicity.
```

> This observation is also consistent with preliminary evidence that certain psychotropic drugs may be protective against COVID-19 [@doi:10.1016/j.drudis.2020.06.022; @doi:10.1007/s00406-020-01231-x; @doi:10.1001/jama.2020.22760].

Reviewer 2: ```suggestion
This observation would be consistent with preliminary evidence that certain psychotropic drugs may be protective against COVID-19 [@doi:10.1016/j.drudis.2020.06.022; @doi:10.1007/s00406-020-01231-x; @doi:10.1001/jama.2020.22760].
```

> While _in vivo_ application of this principle has not yet been successful, this effort demonstrates the potential for high-throughput screening to reveal general biological principles that may be beneficial in developing strategies to combat SARS-CoV-2.

Reviewer 2: ```suggestion
However, these effects may not hold _in vivo_.
While clinical application of high-throughput screening for compounds against SARS-CoV-2, this effort demonstrates its potential to reveal general biological principles that may be beneficial in developing strategies to combat SARS-CoV-2.
```

Other large-scale efforts to screen candidate drugs against SARS-CoV-2 (e.g., [@doi:10.1016/j.celrep.2021.108959]) may complement these efforts and provide additional insights into the fundamental characteristics of drugs that show potential for repurposing.

> <!-- I read the Tummino et al. study, and this isn't how I woudl characterize their findings. My understanding is that they hypothesize that these cationic amphiphilic drugs induce phospholipidosis, which has antiviral activity in in vitro assays but none in vivo. They are characterizing these compounds as confounders or false positives that were detected in the computational or experimental screens and define chemical structure-based rules to identify similar false positives. The current framing seems to imply that cationic amphiphilic drugs may still have clinical success, but the paper argues that pursuing these confounders (including CQ/HCQ) has been a big waste of time and money in clinical trials. I read the journal version, not the preprint, so the message may have changed. -->

Reviewer 2: Thank you so much for catching this! I read it a while ago and I don't remember the details at all. Would the comments above address your concerns?

Author: I think so.  I'll commit all your changes and then read through the paragraph again.

Author: I made additional changes in 3dcd7e2a93d9c4bc0ab2078d5d664fa0325900b8 to match my interpretation of Tummino et al.  How does that look to you?

Once again, this approach does not require the development of compounds specifically designed for the emergent viral threat, but rather hinges on a theoretical understanding of how the virus interacts with a human host and how pharmaceuticals can be used to modify those interactions.
In contrast, other mAbs can be designed based on a specific pathogen.
COVID-19 has demonstrated that this development can progress rapidly, with two mAb cocktails currently approved in the United States.

> <!-- Do we have any broad insights into the types of trials that have been run and the degree of clinical evidence they provide? It seems as if in several cases there were small initial trials (some observational or underpowered) that didn't provide strong evidence for or against a drug. We had to wait for RECOVERY, Solidarity, etc. results to draw better conclusions. Is that accurate or worth commenting on? -->

Reviewer 2: I'll work on opening a PR to address this and the comment below!

Author: Great!  We can leave these comments here for now and resolve them in a future PR after merging this one, which already touches many files.

Reviewer 2: The more I think about this comment, the more I think this is really the crux of the paper.
